site/src/components/BuildsTable/BuildsTable.tsx

```diff
@@ -26,9 +26,11 @@ export const Language = {
 export interface BuildsTableProps {
   builds?: TypesGen.WorkspaceBuild[]
   className?: string
+  username: string
+  workspaceName: string
```

Comment:
We shouldn't need to pass these as props to build the URL. Instead, we can make use of[ React Router's `useParams()` hook](https://reactrouter.com/docs/en/v6/hooks/use-params), since these segments are already available in the URL for this page. You can look at an example in `CreateWorkspacePage.tsx`

Reply:
Thank you, the code is cleaner now! 🎉